_pages/dev/styling-and-page-layout/styling-and-page-layout.md

```diff
@@ -4,7 +4,8 @@ title: Styling and Page Layout
 
 This is a landing page for grouping together topics about styling and page layout. These topics include the following:
 
-- [Global Variables]({{ site.baseurl }}{% link _pages/dev/styling-and-page-layout/global-variables.md %})
 - [CSS Architecture]({{ site.baseurl }}{% link _pages/dev/styling-and-page-layout/css-architecture.md %})
+- [Directionality]({{ site.baseurl }}{% link _pages/dev/styling-and-page-layout/directionality.md %})
```

Comment:
same here, I'd move directionality below page layout. It's less important.

Reply:
Fixed 🙂